shared/reducers/settings.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as CommonConstants from '../constants/common'
+import * as Constants from '../constants/settings'
+import type {Actions, State} from '../constants/settings'
+
+const initialState: State = {
+  notifications: {
+    settings: null,
+    unsubscribedFromAll: null,
+  },
+}
+
+function reducer (state: State = initialState, action: Actions) {
+  switch (action.type) {
+    case CommonConstants.resetStore:
+      return {...initialState}
+    case Constants.notificationsRefreshed:
+      if (action.error) { break }
```

Comment:
FYI just to note here, I don't think this line is reachable atm since we never create `notificationsRefreshed` with an error